src/components/UsageReportConsent.js

```diff
@@ -0,0 +1,47 @@
+import { h } from 'preact'
+import { useState, useRef, useEffect } from 'preact/hooks'
+import { useNavigation, useI18n, useSoftkey } from 'hooks'
+import { RadioListView } from 'components'
+import { getConsentStatus, setConsentStatus } from 'utils'
+
+export const UsageReportConsent = ({ close }) => {
+  const i18n = useI18n()
+  const containerRef = useRef()
+  const listRef = useRef()
+  const hasConsented = getConsentStatus()
+  const [items, setItems] = useState([{ title: i18n('usage-consent-yes'), isSelected: hasConsented }, { title: i18n('usage-consent-no'), isSelected: !hasConsented }])
+  const [, setNavigation, getCurrent] = useNavigation('UsageReportConsent', containerRef, listRef, 'y')
+
+  const onKeyCenter = () => {
+    const { index } = getCurrent()
+    setItems(items.map((item, i) => {
+      i === index ? item.isSelected = true : item.isSelected = false
```

Comment:
This can be written as `item.isSelected = (i === index)`. The parens are not even needed but they may add clarity.